File: medilabo-diabetes/src/main/java/com/medilabo/diabetes/service/DiabetesService.java
package com.medilabo.diabetes.service;

import com.medilabo.diabetes.model.entity.Patient;
import com.medilabo.diabetes.model.entity.RiskLevel;

public interface DiabetesService {

    Patient getPatient(String patientId, String token);

    RiskLevel getRisk(String patientId, String token);

}

File: medilabo-frontend/service/diabetes.service.ts
import {base_url} from "@/config/service.config";
import {RiskLevelResponseSchema} from "@/model/response/diabetes.response";
import {api} from "@/config/ky.config";

const endpoint: string = `${base_url}/medilabo-diabetes/risk`;

export const getRisk = async (id: string) => {
    return await api.get(`${endpoint}/${id}`).json().then(RiskLevelResponseSchema.parse);
}